src/screens/SettingsSheet/components/SettingsSection.tsx

```diff
@@ -134,16 +133,9 @@ const SettingsSection = ({
   const onPressReview = useCallback(async () => {
     if (ios) {
       onCloseModal();
-      RainbowRequestReview.requestReview((handled: boolean) => {
-        if (!handled) {
-          AsyncStorage.setItem(REVIEW_DONE_KEY, 'true');
-          Linking.openURL(AppleReviewAddress);
-        }
-      });
-    } else {
-      RNReview.show();
     }
```

Comment:
Does this need to be iOS specific?

Reply:
probably not, this is probably legacy bs from pre android release but worth double checking